packaging/prover-proxy/miden-prover-proxy.service

```diff
@@ -7,7 +7,8 @@ WantedBy=multi-user.target
 
 [Service]
 Type=exec
-ExecStart=/usr/bin/miden-proving-service start-proxy --port 50051 10.0.1.254:50052
+EnvironmentFile=/etc/opt/miden-prover-proxy/service.conf
+ExecStart=/usr/bin/miden-proving-service start-proxy --port ${PORT} ${PROVER_HOST}:${PROVER_PORT}
```

Comment:
Not from this PR, but `--port` parameter is not mentioned in the [docs](https://github.com/0xMiden/miden-base/tree/next/bin/proving-service#proxy). We should probably add it there.